deploy/cr.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: pg.percona.com/v2beta1
 kind: PerconaPGCluster
 metadata:
   name: cluster1
+#  finalizers:
+#  - pg.percona.com/delete-pvc
```

Comment:
Finalizers in our other operators are named without the domain. Is it necessary to have it?